server/controllers/memoCtrl.js

```diff
@@ -101,18 +113,26 @@ async function renderMemoEditorPage(req, res, next) {
      */
 
     // start
-    const apiMemoDataDeepCopy = JSON.parse(JSON.stringify(apiMemoData))
+    const apiMemoDataDeepCopy = apiMemoData
     const { applicationCodes } = apiMemoDataDeepCopy
 
+    // fix this so there's only one function
```

Comment:
This should still be fixed

Reply:
It was lost in the merge apparently. Looking at it now